packages/ui-kit/src/components/PieChart/PieChart.tsx

```diff
@@ -0,0 +1,326 @@
+            display: true,
+            labels: {
+              usePointStyle: true,
+              pointStyle: '*',
+              pointStyleWidth: 8,
+              boxHeight: 6
+            }
+          },
+          customChartLabelsPlugin
+        }
+
+        const datasets = isDoughnut ? { cutout: '75%' } : { cutout: '0' }
+
+        if (chartRef.current) {
+          const chart = chartRef.current
+
+          chart.data.labels = labels
+          Object.assign(chart.data.datasets[0], {
+            type: variant,
+            data: values,
+            backgroundColor: chartColorPlatte,
+            ...datasets
+          })
+
+          chart.options.plugins = {
+            ...chart.options.plugins,
+            ...customPlugins
+          }
+
+          chart.update()
+          return
+        }
+
+        let config: ChartConfiguration<'pie' | 'doughnut'> = {
+          ...chartConfig,
+          type: variant,
+          data: {
+            labels: labels,
+            datasets: [
+              {
+                data: values as number[],
+                backgroundColor: chartColorPlatte,
+                borderWidth: 0,
+                offset: 4,
+                hoverOffset: 20,
+                ...datasets
+              }
+            ]
+          },
+          options: {
+            ...chartConfig.options,
+            indexAxis: 'y',
+            responsive: true,
+            maintainAspectRatio: false,
+            layout: {
+              padding: 4
+            },
+            plugins: {
+              ...chartConfig.options?.plugins,
+              ...customPlugins
+            },
+            scales: {
+              x: {
+                display: false
+              },
+              y: {
+                display: false
+              }
+            }
+          },
+          plugins: [customCanvasBackgroundColor, customChartLabelsPlugin]
+        }
+
+        if (chartConfigProps) {
+          config = chartConfigProps(config)
+        }
+
+        chartRef.current = new ChartJS(canvasRef.current, config) as ChartJS
+        canvasRef.current.style.borderRadius = '0px'
+      },
+      [variant, theme, card, chartProps, chartConfig, isDoughnut, chartConfigProps]
+    )
+
+    const destroyChart = () => {
+      if (chartRef.current) {
+        chartRef.current.destroy()
+        chartRef.current = null
+      }
+    }
+
+    // Calculate the other value and add it to the leaderboardData
+    const fetchedData = React.useMemo(() => {
+      if (!leaderboardData || !counterData) {
+        return
+      }
+
+      let leaderboardSum = 0
+
+      for (const row of leaderboardData?.leaderboard?.rows ?? []) {
+        leaderboardSum += parseFloat(row[1] as string)
+      }
+
+      const otherValue = parseFloat(counterData?.counter?.value ?? '0') - leaderboardSum
+      leaderboardData?.leaderboard?.rows.push(['Other', otherValue.toString()])
+
+      return leaderboardData
+    }, [leaderboardData, counterData])
```

Comment:
If we create the `usePie` hook we can move this logic there

Reply:
not need to create a usePie since we are going to call only useLeaderboard hook when we extend it with total value.